production/helm/loki/ci/default-values.yaml

```diff
@@ -6,3 +6,11 @@ read:
   replicas: 1
 write:
   replicas: 1
+monitoring:
+  serviceMonitor:
+    labels:
+      release: "prometheus"
+  selfMonitoring:
+    lokiCanary:
+      test:
+        prometheusAddress: "http://prometheus-kube-prometheus-prometheus.prometheus.svc.cluster.local:9090"
```

Comment:
Is this an option for the Canary?

Reply:
For the test. It needs to know which prometheus to query the canary's metrics from.